Utils/Dataflow/095_datasetInfoAMI/amiDatasets.py

```diff
@@ -23,18 +23,23 @@
     sys.exit(1)
 
 ami_client = None
-PHYS_VALUES = [{"ami": "genFiltEff", "es": "gen_filt_eff"},
-               {"ami": "crossSection", "es": "cross_section"},
-               {"ami": "crossSectionRef", "es": "cross_section_ref"},
-               {"ami": "kFactor", "es": "k_factor"},
-               {"ami": "processGroup", "es": "process_group"},
-               {"ami": "mePDF", "es": "me_pdf"},
+# Field names in terms of AMI and ES schemes.
+PHYS_VALUES = [{'ami': 'genFiltEff', 'es': 'gen_filt_eff'},
```

Comment:
Better to keep empty line between `ami_client` and `PHYS_VALUES`, for without one it looks like piece of code with  logically linked lines -- or group of independant, but similar one-line statements. Which it is not: both `PHYS_VALUES` and `ami_client` are, of course, global variables, but they have different purpose:
* `PHYS_VALUES` contains some reference data, required for specific logical operation. Its purpose id to move this data from code, making the latter more compact and avoiding the variable re-definition every time the function is called;
* `ami_client` stores a commonly used object. It is defined dynamically during the program execution and was introduced to avoid passing the object to every other function as parameter.

Also, these statemets are not one-liners -- so to group them as "similar independant statements", there should be a bit more than absence of blank line: e.g. common header ("Global variables definition") and unified descriptions for all variables defined in this block. Still I believe blank line would be good for readability, preventing things from bluring together. Example below is just to demonstrate what I'm talking about -- it is a bit tooo massive for a couple of variables, so the code becomes much less compact and may lose more, than gain.
```
# Global variables --->
# ---

# AMI client
ami_client = None

# Field names in terms of AMI and ES schemes
PHYS_VALUES = [{'ami': ..., 'es': ...},
               {'ami': ..., 'es': ...},
               {'ami': ..., 'es': ...},
               {'ami': ..., 'es': ...},
               {'ami': ..., 'es': ...}
               ]

# Allowed dataset formats, for which AMI should be queried
FILTER = ['AOD', 'EVNT', 'HITS']

# <--- Global variables
```

---

*Started to look into the PR a bit too early, so maybe it was gonna be changed later. But this part looks the same way in #282, so I decided not to remove the comment (since it was already written when I realized that I am too hasty ;) ).*

Reply:
Good point, done.

> _Started to look into the PR a bit too early, so maybe it was gonna be changed later. But this part looks the same way in #282, so I decided not to remove the comment (since it was already written when I realized that I am too hasty ;) )._

Not, it was not planned. It's ok, thanks for the suggestion.